src/Raven.Server/Documents/DatabaseRequestHandler.cs

```diff
@@ -73,7 +73,7 @@ protected DisposableAction TrackRequestTime(string source, bool doPerformanceHin
                     Database
                         .NotificationCenter
                         .RequestLatency
-                        .AddHint(HttpContext.Request.Path, sw.ElapsedMilliseconds, source);
+                        .AddHint(HttpContext.Request.Path,HttpContext.Request.Query, sw.ElapsedMilliseconds, source, HttpContext.Request.ContentLength ?? 0);
```

Comment:
we are using POST now for queries, so Request.Query will not have any good info, there is an issue for that:  http://issues.hibernatingrhinos.com/issue/RavenDB-9201